js/private/js_run_devserver.mjs

```diff
@@ -96,7 +118,8 @@ async function syncRecursive(src, dst, writePerm) {
                 // Intentionally synchronous; see comment on mkdirpSync
                 mkdirpSync(path.dirname(dst))
             }
-            await fs.promises.symlink(src, dst)
+            const symlinkArgs = getSymlinkArgs(src);
+            await fs.promises.symlink(src, dst, ...symlinkArgs)
```

Comment:
Can we change the `getSymlinkArgs` result to be a single value instead of array? What would the default be for non-windows?